Additional info on changing is_focused through focus_element (#1446)
browserengineering/book: book/accessibility.md

```

Set this `is_focused` flag in a new `focus_element` method that we'll now use

> to change the `focus` field in a `Tab`. Update existing code that sets

Reviewer: ```suggestion
to change the `focus` field in a `Tab` (and update existing code that sets
```

> to change the `focus` field in a `Tab` (and update existing code that sets

Reviewer: ```suggestion
to change the `focus` field in a `Tab` (you should also update existing code that sets
```

> `is_focused` directly to call focus_element instead):

Reviewer: ```suggestion
`is_focused` directly to call `focus_element` instead):
```
